apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1076,6 +1069,48 @@ export class InstallManager {
     });
   }
 
+  private static _mergeEnvironmentVariables(
+    baseEnv: NodeJS.ProcessEnv,
+    environmentVariables: { [environmentVariableName: string]: IEnvironmentVariable; } | undefined
+  ): NodeJS.ProcessEnv {
+    const packageManagerEnv: NodeJS.ProcessEnv = baseEnv;
+
+    if (environmentVariables) {
+      for (const envVar in environmentVariables) {
+        if ({}.hasOwnProperty.call(environmentVariables, envVar)) {
```

Comment:
```suggestion
        if (environmentVariables.hasOwnProperty(envVar)) {
```

If that doesn't work, just remove the check and suppress the lint error. Our JSON parser doesn't populate `hasOwnProperty` for some reason.

Reply:
The suggested change doesn't work. I will suppress the lint error.